addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -95,6 +102,20 @@ def _compute_state(self):
             elif all(picking.state in ['cancel', 'done'] for picking in batch.picking_ids):
                 batch.state = 'done'
 
+    @api.depends('picking_ids.scheduled_date')
```

Comment:
missing picking_ids

Reply:
Oops, added it in

Reply:
Apparently it's not needed anymore. Adding a picking will trigger the compute. Anyway you can let it.